src/react-components/alert.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+class Alert extends React.Component {
+  constructor(props){
+    super(props);
+  }
+  render() {
+    const classes = `alert ${this.props.class}`;
+    return (
+      <div class={classes} role="alert">
```

Comment:
class attribute interpreted by react should be className. Like this`<div className={classes} role="alert">`